sefaria/settings_utils.py

```diff
@@ -0,0 +1,31 @@
+from http.client import HTTPException
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+import os
+
+def init_sentry(SENTRY_DSN):
+
+    def before_send(event, hint):
+        # Check if the event has an exception
+        if 'exc_info' in hint:
+            exc_type, exc_value, tb = hint['exc_info']
+            # If it is not an HTTP 500 error, drop the event
+            if isinstance(exc_value, HTTPException) and exc_value.code != 500:
+                return None
+
+        # If there is no exception, drop the event
+        if 'exception' not in event:
+            return None
+
+        return event
+
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        environment=os.getenv("SENTRY_ENVIRONMENT", "unknown"),
```

Comment:
Here we should be referencing the variables out of local settings, not the envvars directly. Parsing them from envvars should be done in uniformity with the rest, in local_settings

Reply:
this function is being implemented on init in the settings.py file, but I do hear what you are saying - so I expanded the vars it will accept